README.md

```diff
@@ -249,7 +249,13 @@ Note that `BROKER_CLIENT_URL` now has the HTTPS scheme.
 
```

Comment:
If we end up going with `JIRA_HOSTNAME` then make sure to update line 190 above

Reply:
Totally!